clang/test/CXX/drs/cwg28xx.cpp

```diff
@@ -67,6 +63,27 @@ void B<int>::g() requires true;
 
 } // namespace cwg2847
 
+namespace cwg2857 { // cwg2857: 2.7
+struct A {};
+struct B {
+  int operator+(A);
+};
+template <typename>
+struct D;
+
+void f(A* a, D<int>* d) {
+  *d + *a;
```

Comment:
This wouldn't be an ADL lookup since member functions can't be found via ADL. It looks like clang doesn't actually implement CWG2857: https://godbolt.org/z/5xdYj7a35 / https://godbolt.org/z/7oqfE5WKW

Reply:
Thank you! I used one of your tests instead, and stopped claiming that CWG2857 is available.